pyproject.toml

```diff
@@ -108,6 +108,16 @@ visual_prompt = [
     "scikit-image",
 ]
 
+detection = [
+    "mmpretrain==1.0.*",
+    "mmdet==3.1.0",
+    "mmdeploy",
+    "timm==0.6.12",
+    "pytorchcv",
+    "yapf<0.40.0",
+    "scikit-image",
+]
```

Comment:
Just a question. As we are mainly going for mmX but what if we are adding detectron/ultralytics?

Reply:
Then we might need to branch such as otx install mmdetection otx install detectron, but I'm not sure

Reply:
I agree with @jaegukhyun. This `detection` section will probably become `mmdetection`.